test/scripts/helper.py

```diff
@@ -0,0 +1,19 @@
+from pathlib import Path
+import sys
+import importlib
+
+CWD = Path(__file__).parent.resolve()
+
+
+def import_path(module_name):
+    """Returns the module as object"""
+    # Get the right path to the module
+    path = str(CWD.parent.parent.joinpath('scripts', module_name))
+    spec = importlib.util.spec_from_loader(
+        module_name,
+        importlib.machinery.SourceFileLoader(module_name, path)
+    )
+    module = importlib.util.module_from_spec(spec)
+    spec.loader.exec_module(module)
+    sys.modules[module_name] = module
+    return module
```

Comment:
thanks to python's convoluted import machinery we have to go to these lengths to import modules which don't have a `.py` suffix

Reply:
full disclosure: I also found that the way this can be done changed at least twice in an incompatible way since py3.0 was established.. I do hope this way is a little longer lasting..